### PR TITLE
Fix example statoscope.config.js in stats-validator-plugin-webpack

### PR DESCRIPTION
**Pull request by micro-chipset**

### CODE REVIEW

File: packages/stats-validator-plugin-webpack/README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ This package contains rules to validate a webpack bundle.
           // ensures that the build time has not exceeded the limit (10 sec)
           '@statoscope/webpack/build-time-limits': ['error', 10000],
           // ensures that bundle doesn't use specified modules
-          '@statoscope/webpack/restricted-modules': ['error', [/some-package\/dist/some-module\.js/]],
+          '@statoscope/webpack/restricted-modules': ['error', ["/some-package/dist/some-module.js"]],
```
Comment (smelukov) [inline packages/stats-validator-plugin-webpack/README.md:26]: We need a regexp here, because we don't know entire module path

Comment (micro-chipset) [reply packages/stats-validator-plugin-webpack/README.md]: changed to RegExp

File: packages/stats-validator-plugin-webpack/README.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ This package contains rules to validate a webpack bundle.
           // ensures that the build time has not exceeded the limit (10 sec)
           '@statoscope/webpack/build-time-limits': ['error', 10000],
           // ensures that bundle doesn't use specified modules
-          '@statoscope/webpack/restricted-modules': ['error', [/some-package\/dist/some-module\.js/]],
+          '@statoscope/webpack/restricted-modules': ['error', [/some-module\.js/,]],
```
Comment (smelukov) [inline packages/stats-validator-plugin-webpack/README.md:26]: The main thing that I wanted to show by this example is that we can restrict using a specific module from any package and not restrict a whole package
For example `/\/lodash\/dist\/min\.lodash\.js/`

Comment (micro-chipset) [reply packages/stats-validator-plugin-webpack/README.md]: @smelukov Thank you for the clarification. See it all right?

**Comment by smelukov** [comment]
Thanks

**Pull request merged**